public/profile/profile.js

```diff
@@ -40,7 +40,7 @@ angular.module('VBattle.profile', [])
   $scope.editAvatar = function () {
     var avatar = {
       "avatarData": {
-        "image": this.value.image,
+        "imageSource": this.value.imageSource,
```

Comment:
hey zach, this is where imageSource is changed for EDIT avatar. not ADDIT avatar. Thank you very much